src/Pages/ReferenceTokens.razor

```diff
@@ -87,6 +88,13 @@
     
     private void CheckReferenceToken(EditContext editContext)
     {
+        Dictionary<string, string> Data = new()
```

Comment:
As `Data` is just a local variable here we should adhere to the convention of lowercase variable names.

Reply:
changed in 5a020fb